### DISCUSSION TITLE
Backup plugin

### DISCUSSION THREAD

**Discussion by ctuxboy** [BoidCMS/BoidCMS#43] [Ideas]
Short question: it would be helpful if there was a backup option or plugin 🙈

**Comment by sysadev** [comment]
Hello @ctuxboy, I'm so so happy to tell you that a backup plugin was made about two months ago. I just never had the right time to make it available. But I'll make it available soon.

**Reply by ctuxboy** [reply]
WOW!!!!
AWESOME!
After making changes it is more useful to quickly make a backup with a one click than 'transfer' via an FTP client.
Thanks a lot man 🙏
